datasets/titanic/titanic_by_openml_converter.json

```diff
@@ -1,124 +1,121 @@
 {
   "@context": {
     "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
     "ml": "http://mlcommons.org/schema/"
   },
+  "@type": "sc:Dataset",
   "@language": "en",
-  "@type": "Dataset",
-  "creativeWorkStatus": "active",
+  "name": "Titanic",
+  "description": "**Author**: Frank E. Harrell Jr., Thomas Cason  \n**Source**: [Vanderbilt Biostatistics](http://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic.html)  \n**Please cite**:   \n\nThe original Titanic dataset, describing the survival status of individual passengers on the Titanic. The titanic data does not contain information from the crew, but it does contain actual ages of half of the passengers. The principal source for data about Titanic passengers is the Encyclopedia Titanica. The datasets used here were begun by a variety of researchers. One of the original sources is Eaton & Haas (1994) Titanic: Triumph and Tragedy, Patrick Stephens Ltd, which includes a passenger list created by many researchers and edited by Michael A. Findlay.\n\nThomas Cason of UVa has greatly updated and improved the titanic data frame using the Encyclopedia Titanica and created the dataset here. Some duplicate passengers have been dropped, many errors corrected, many missing ages filled in, and new variables created. \n\nFor more information about how this dataset was constructed:\nhttp://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic3info.txt\n\n\n### Attribute information  \n\nThe variables on our extracted dataset are pclass, survived, name, age, embarked, home.dest, room, ticket, boat, and sex. pclass refers to passenger class (1st, 2nd, 3rd), and is a proxy for socio-economic class. Age is in years, and some infants had fractional values. The titanic2 data frame has no missing data and includes records for the crew, but age is dichotomized at adult vs. child. These data were obtained from Robert Dawson, Saint Mary's University, E-mail. The variables are pclass, age, sex, survived. These data frames are useful for demonstrating many of the functions in Hmisc as well as demonstrating binary logistic regression analysis using the Design library. For more details and references see Simonoff, Jeffrey S (1997): The \"unusual episode\" and a second statistics course. J Statistics Education, Vol. 5 No. 1.",
```

Comment:
Are "author", "source", etc. part of the original description in markdown? Or do you generate that mark-down from your metadata? If it's the latter, then it would make sense to output the corresponding schema.org fields instead. If the former, I think it's fine to leave as is. (We don't want to start parsing mark-down to extract these fields.)

Reply:
Unfortunately, this is part of the original description. The person who created this dataset had the option to put the author in the `author` field, but decided not to.